Source/Krypton Components/Krypton.Toolkit/Controls Visuals/VisualTaskDialog.cs

```diff
@@ -1432,7 +1432,7 @@ private void InitializeComponent()
             // 
             this._footerLabel.AutoSize = false;
             this._footerLabel.Font = new System.Drawing.Font("Segoe UI", 9F);
-            this._footerLabel.ForeColor = System.Drawing.Color.White;
+            this._footerLabel.ForeColor = System.Drawing.Color.FromArgb(((int)(((byte)(30)))), ((int)(((byte)(57)))), ((int)(((byte)(91)))));
```

Comment:
Theme Colours....

Reply:
It does, but the designer code updates when the theme is changed. Should the `ForeColor` be serialised, i.e. hidden?

Reply:
Try running these things on a Windows Light them and see what happens.
Default colours in the designer need to be form the the selected theme.
Forcing these overrides is confusing, thats why the other controls have "Init from theme" calls
If they are not to be set separately then they should be not be visible in the designer serialisation. as these things should just be controls (Like the rest of them in the KMessageBox)

Reply:
@Smurf-IV I'm going to recreate the `VisualTaskDialog`, as at the moment everything (designer & logic code) is 'shoved' into one huge class.